azuredeploy.json

```diff
@@ -1,18 +1,18 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.1",
     "parameters": {
         "scopeID": {
-            "type": "String"
+            "type": "string"
         },
         "iotCentralSASKey": {
-            "type": "String"
+            "type": "string"
         }
     },
     "variables": {
-        "planName": "[concat('iotc-pln', uniqueString(resourceGroup().id))]",
+        "planName": "[concat('iotcpln', uniqueString(resourceGroup().id))]",
```

Comment:
Don't change the naming layout. This change is just a refactor.

Reply:
emmm okay, I feel its inconsistent.